naxsi_src/naxsi_net.c

```diff
@@ -76,5 +72,4 @@ is_in_subnet(const cidr_t* cidr, const ip_t* ip, int is_ipv6)
     return (ip->v6[0] & cidr->mask.v6[0]) == (cidr->subnet.v6[0] & cidr->mask.v6[0]) &&
            (ip->v6[1] & cidr->mask.v6[1]) == (cidr->subnet.v6[1] & cidr->mask.v6[1]);
   }
-  return 0;
```

Comment:
remove the else and just put the return above as the final return.

Reply:
If I understand this correctly, added the change in [99917e4](https://github.com/wargio/naxsi/pull/42/commits/99917e4a6af1eca8c1aed438d33a7155e37c8342).